magma/home/__init__.py

```diff
@@ -9,7 +9,7 @@
 except ImportError:
     from io import StringIO
 
-from flask import Blueprint, request, render_template, make_response
+from flask import Blueprint, flash, request, render_template, make_response, session
```

Comment:
@gravesm this line is a bit long but wasn't sure if you prefer the extra line for 4 chars over 80 or not.